lib/http_negotiate.c

```diff
@@ -143,7 +143,13 @@ CURLcode Curl_output_negotiate(struct connectdata *conn, bool proxy)
     }
     if(!neg_ctx->context) {
       result = Curl_input_negotiate(conn, proxy, "Negotiate");
-      if(result)
+      if(result == CURLE_LOGIN_DENIED) {
+        /* negotiate auth failed, let's continue unauthenticated to stay
+         * compatible with the behavior before curl-7_64_0-158-g6c6035532 */
```

Comment:
What's curl-7_64_0-158-g6c6035532? Some RedHat'ism? Maybe mention the commit instead?

Reply:
I believe that `curl-7_64_0-158-g6c6035532` is an upstream commit -- see `https://github.com/curl/curl/commit/curl-7_64_0-158-g6c6035532`.  I got it as the output of `git describe`.  The advantage of this format is that it immediately tells you the last upstream release that does not contain the commit (without even having git installed, or having network availability).

Reply:
Ah, interesting, I didn't know about that format, thanks!